### PR TITLE
Add import support for google_compute_image.

### PR DESCRIPTION
**Pull request by rosbo**
issue #182 
/cc @danawillow

### CODE REVIEW

File: google/resource_compute_image.go [hunk 1]
```diff
@@ -195,3 +213,9 @@ func resourceComputeImageDelete(d *schema.ResourceData, meta interface{}) error
 	d.SetId("")
 	return nil
 }
+
+func resourceComputeImageImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	d.Set("create_timeout", computeImageCreateTimeoutDefault)
```
Comment (danawillow) [inline google/resource_compute_image.go]: Can you explain why this is necessary instead of setting `create_timeout` on read every time?

Comment (rosbo) [inline google/resource_compute_image.go]: Added a comment explaining why.

Another option would be to set the `create_timeout` in the read method if it is not set but I prefer not cluttering the read method logic with logic specific to import.

Comment (danawillow) [inline google/resource_compute_image.go]: Hmm, so actually I think we should stop this from being ForceNew. Just from thinking about it logically, it's the amount of time we're willing to wait for the instance to create. IMO, changing it doesn't actually mean I want to create a new one, it just means that should it recreate for some other reason, we would want to wait that long. Does that seem right? If so we can remove ForceNew and just never do a `d.Set` on it.

Comment (rosbo) [inline google/resource_compute_image.go]: Agree for the ForceNew part, I will do it.

However, if we don't set `create_timeout` to the default value on import, then, after importing, the user will always have a "plan diff" with:
  ~ google_compute_image.web-image
      create_timeout: "" => "4"

The update is fast (only update the value locally). I am wondering if this is the expected behavior that after an update, we may have a diff that we can't fix without either running apply or fiddling with the state file.

Thoughts?

Comment (danawillow) [inline google/resource_compute_image.go]: Oh, you're right. In this case the way you've done it is probably correct, then.

Comment (rosbo) [inline google/resource_compute_image.go]: I removed the ForceNew from the create_timeout field.

File: website/docs/r/compute_image.html.markdown [hunk 2]
```diff
@@ -83,3 +83,11 @@ In addition to the arguments listed above, the following computed attributes are
 exported:
 
 * `self_link` - The URI of the created resource.
+
+## Import
+
+VM image can be imported using the `name`, e.g.
+
+```
+$ terraform import google_compute_image.web-image image1
```
Comment (danawillow) [inline website/docs/r/compute_image.html.markdown]: Can you change `image1` to `my-custom-image` since that's the name in the example?

Comment (rosbo) [inline website/docs/r/compute_image.html.markdown]: Done

**Comment by danawillow** [comment]
Looks good, thanks!

```
make testacc TEST=./google TESTARGS='-run=TestAccComputeImage'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./google -v -run=TestAccComputeImage -timeout 120m
=== RUN   TestAccComputeImage_resolveImage
--- PASS: TestAccComputeImage_resolveImage (83.84s)
=== RUN   TestAccComputeImage_importFromRawDisk
--- PASS: TestAccComputeImage_importFromRawDisk (67.65s)
=== RUN   TestAccComputeImage_importFromSourceDisk
--- PASS: TestAccComputeImage_importFromSourceDisk (80.38s)
=== RUN   TestAccComputeImage_basic
--- PASS: TestAccComputeImage_basic (77.72s)
=== RUN   TestAccComputeImage_basedondisk
--- PASS: TestAccComputeImage_basedondisk (81.10s)
PASS
ok  	github.com/terraform-providers/terraform-provider-google/google	390.833s
```

**Pull request merged**